resources/sass/app-ltr.scss

```
@@ -274,6 +274,21 @@ ul {
   }
 }
 
+.pretty {
+  white-space: inherit;
+  &.form-check-input {
+    margin-left: 0;
+  }
+  .state label{
+    text-indent: 0;
+    padding-left: 2rem;
+    line-height: 1.4em;
+    &:after, &:before {
+      top: 0;
+    }
+  }
+}
+
```

That's great!

However, we will have to adapt it so it works for RTL direction too:
```suggestion
.pretty {
  white-space: inherit;
  &.form-check-input {
    @if $htmldir == ltr {
      margin-left: 0;
    } @else {
      margin-right: 0;
    }
  }
  .state label{
    text-indent: 0;
    @if $htmldir == ltr {
      padding-left: 2rem;
    } @else {
      padding-right: 2rem;
    }
    line-height: 1.4em;
    &:after, &:before {
      top: 0;
    }
  }
}

```

---

Thank you @asbiin for the suggestion.
I've made the changes.